src/openapi/streaming/streaming.ts

```diff
@@ -167,6 +168,7 @@ class Streaming extends MicroEmitter<EmittedEvents> {
     disposed = false;
     private heartBeatLog: Array<[number, ReadonlyArray<string>]> = [];
     shouldSubscribeBeforeStreamingSetup = false;
+    onProbe?: (message: types.ProbeControlMessage) => void;

```

Comment:
Let's make this private, matching the existing heartbeat stuff

Reply:
Yep, makes sense